index.bs

```diff
@@ -1089,6 +1089,9 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 :: An identifier that is [=human palatability|human-palatable=] is intended to be rememberable and reproducible by typical human
     users, in contrast to identifiers that are, for example, randomly generated sequences of bits [[EduPersonObjectClassSpec]].
 
+: <dfn>Non-Discoverable Credential</dfn>
+:: This is a [=credential=] whose [=credential ID=] must be provided in {{publickeycredentialrequestoptions/allowCredentials}} when calling {{CredentialsContainer/get()|navigator.credentials.get()}} because it is not client-side discoverable. See also [=server-side credentials=].
```

Comment:
```suggestion
:: This is a [=credential=] whose [=credential ID=] must be provided in {{publickeycredentialrequestoptions/allowCredentials}} when calling {{CredentialsContainer/get()|navigator.credentials.get()}} because it is not [=client-side discoverable credential|client-side discoverable=]. See also [=server-side credentials=].
```